packages/frontend-2/lib/core/configs/apollo.ts

```diff
@@ -105,6 +105,10 @@ function createCache(): InMemoryCache {
           automateFunctions: {
             keyArgs: ['filter', 'limit'],
             merge: buildAbstractCollectionMergeFunction('AutomateFunctionCollection')
+          },
+          workspaceBySlug: {
+            keyArgs: ['slug'],
+            merge: mergeAsObjectsFunction
```

Comment:
we didn't usually need a custom merge function for non-array/collection fields, is this really needed?